README.md

```diff
@@ -9,7 +9,7 @@ A Debian-based system
 ## Role Variables
 
 | Name                            | Required/Default          | Description                                                                                                                     |
-|:--------------------------------|:--------------------------|:--------------------------------------------------------------------------------------------------------------------------------|
+| :------------------------------ | :------------------------ | :------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
This is an unrelated change due to use of different autoformatters. Use `git add -p`.